### DISCUSSION TITLE
解凍と呼ぶか、展開と呼ぶか

### DISCUSSION THREAD

**Discussion by yoshi389111** [Qithub-BOT/QiiCipher#59] [一般]
現在 `bin/archive` `bin/dearchive` コマンド、 `README.md` で「解凍」という言葉が使われています。

個人的には気にならない世代ですが、最近（？）は「展開」が主流だという説もあります。

* 参考： [“解凍”と呼ぶのはおっさんだけ？　Twitterで“解凍”派と“展開”派の争いが勃発](https://forest.watch.impress.co.jp/docs/serial/yajiuma/759537.html) 2016年の記事

LHA 方言だと言われると、そうかもなーとも思います。
修正すべきでしょうか？

**Comment by KEINOS** [comment]
「巻き戻し」が伝わらないのと同じやつですね！

私も習慣的に「圧縮」→「解凍」世代なので気にならないのですが、確かに「解凍」なら「冷凍」と対になっていないと変だし、冷凍ってのも変。

内容的には "archive" を「圧縮」というのは変なのと同じ気がします。そして、一度気になると、とても気になるとこも！w

両方を取って「アーカイブを解凍する」→「圧縮アーカイブを展開／解凍する」とするのはどうでしょうか。
